tapis-project/tapis-ui
# task/TUI-195 - file browsing

## Overview:

Add support in the file browser for entering and leaving folders

## Related Github Issues:

- [TUI-195](https://github.com/tapis-project/tapis-ui/issues/195)

## Summary of Changes:

- Update tapis-typescript
- Conditionally render file items as directories or files
- Clicking on files triggers onNavigate callback
- tapis-app filelisting routes to correct path using onNavigate callback

## Testing Steps:

1. Log in as service account
2. Browse through folders
3. Make sure you can traverse back to parent folder

## UI Photos:

## Notes:

- Requires unit tests
